tests/json/schema-1-tamps/input.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import psi4
+import json
+
+#Generate JSON data
+json_data = {
+    "schema_name": "qcschema_input",
+    "schema_version": 1,
+    "return_output": True,
+    "molecule": {
+        "geometry": [
+            0.0,
+            0.0,
+            -0.1294769411935893,
+            0.0,
+            -1.494187339479985,
+            1.0274465079245698,
+            0.0,
+            1.494187339479985,
+            1.0274465079245698
+        ],
+        "symbols": [
+          "O",
+          "H",
+          "H"
+        ],
+        "connectivity" : [
+            (0, 1, 1.0),
+            (0, 2, 1.0)
+        ]
+    },
+    "driver": "energy",
+    "model": {
+        "method": "MP2",
+        "basis" "6-31g"
+    },
+    "keywords": {
+        "scf_type": "df",
+        "mp2_type": "df"
+    },
+    {
+        "extras": {
+            "qcvars": {
+                "tIJAB": {}
+                }
+            }
+        }
+}
+json_ret = psi4.json_wrapper.run_json(json_data)
+
+h2o_test = psi4.geometry("""
+0 1
+O 0.0 0.0 0.1294769411935893
+H 0.0 -1.494187339479985 1.0274465079245698
+H 0.0 1.494187339479985 1.0274465079245698
+""")
+e, wfn = psi4.energy("mp2/6-31g", return_wfn = True)
+tIJAB = wfn.get_amplitudes("tIJAB").to_array
+
+with open("output.json", "w") as ofile:
```

Comment:
I don't think you need to save the return dict to file for the test

Reply:
it's sometimes helpful to do just to check that nothing non-json-able has gotten added to the data structure.